examples/static-file-server/src/tests.rs

```diff
@@ -0,0 +1,64 @@
+use super::*;
+use axum::http::StatusCode as SC;
+use axum::{body::Body, http::Request};
+use http_body_util::BodyExt;
+use tower::ServiceExt;
+
+const INDEX_HTML: &str = include_str!("../assets/index.html");
+const SCRIPT_JS: &str = include_str!("../assets/script.js");
+
+const JS: &str = "text/javascript";
+const HTML: &str = "text/html";
+
+async fn get_page(app: Router, path: &str) -> (StatusCode, String, String) {
```

Comment:
Use `headers::ContentType` instead of `String`.

And I'd argue returning `Bytes` or `Vec<u8>` for content is also better than forcing this to be `String` though I could see an argument for making sure that the returned content is valid UTF-8 before also checking that the contents match though I wouldn't subscribe to that. But feel free to ignore this.